packages/mui/src/stories/decorators/calendarDecorator.tsx

```diff
@@ -1,15 +1,23 @@
 import { DecoratorFunction } from '@storybook/addons';
 import { StoryFnReactReturnType } from '@storybook/react/dist/ts3.9/client/preview/types';
-import { DefaultMuiPickerLocalization } from '../../pickers-common/default-mui-picker-localization';
+import { Locale } from 'src/constant-types';
+import { MultiLocalizationProvider } from 'src/pickers-common';
+import { LocaleWithCustomAdapter } from 'src/pickers-common/MultiLocalizationProvider';
```

Comment:
import from `src/pickers-common`

Reply:
that is ok, I don't want to reexport that variable in its index.ts file!